IM_calculation/scripts/calculate_ims.py

```diff
@@ -206,7 +206,7 @@ def main():
 
     # Create output dir
     utils.setup_dir(args.output_path)
-    if not args.simple_output:
+    if not args.simple_output and args.advanced_ims is None:
```

Comment:
just a note, this was added because the verification on observed workflow is using glob to determine the station list.
so creating a 'stations' folder with nothing in side causes the check_status to always return failed.